spec/features/user_new_account_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.